src/sentry/static/sentry/app/views/projectSettings/pluginNavigation.jsx

```diff
@@ -0,0 +1,45 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Reflux from 'reflux';
+
+import PluginsStore from '../../stores/pluginsStore';
+
+class PluginNavigationItem extends React.PureComponent {
+  static propTypes = {
+    urlRoot: PropTypes.string,
+    id: PropTypes.string,
+    name: PropTypes.string,
+  };
+  render() {
+    let {urlRoot, id, name} = this.props;
+    return (
+      <li>
+        <a href={`${urlRoot}/plugins/${id}/`}>{name}</a>
```

Comment:
I'd inline this - don't think it's worth extracting a component for just this line and the enclosing <li>

Reply:
It's in a sep component so each row can be a PureComponent, I don't think its a source for any performance issues with or without. If you think it's too noisy still I can inline it.

Reply:
Since it's essentially a one liner that isn't reused elsewhere, I think marking it as a PureComponent isn't super valuable, for me it's just a little bit harder to read.